catalog/gitops/configsync/README.md

```diff
@@ -14,6 +14,8 @@ After installing this blueprint, you will be able to:
 3.  Automatically apply your config changes to your Config Controller
     cluster on merge
 
+NOTE: cluster-name must not exceed 25 characters in length
```

Comment:
Can we add a small rationale as to why not?

Reply:
SG